tests/tck/features/lookup/LookUp.IntVid.feature

```diff
@@ -1,104 +1,5 @@
 Feature: LookUpTest_Vid_Int
 
-  Scenario: LookupTest IntVid SimpleVertex
```

Comment:
Why delete these cases?

Reply:
Share these scenarios in another feature file for spaces with different vid types.